config/cypress/cypress.json

```diff
@@ -0,0 +1,11 @@
+{
+   "baseUrl": "http://localhost:8001",
+   "fixturesFolder": "../../tests/fixtures",
+   "ignoreTestFiles": "**/*.png",
+   "integrationFolder": "../../tests/visual",
+   "pluginsFile": "./plugins/index.js",
+   "supportFile": "./support/index.js",
+   "trashAssetsBeforeRuns": true,
```

Comment:
Just to confirm, doing this will not affect the `git diff`, correct? Ie. a screenshot that doesn't change will not show a diff right? (as opposed to one file being deleted and another file being added)

Reply:
correct